conf/application.conf

```diff
@@ -141,3 +141,9 @@ microservice {
   }
 
 }
+
+xsd {
+    goodsReleased = "/xsd/CC025A.xsd"
+}
```

Comment:
Anything that's in here is overwritable, but we don't want these values to be overwritten in configs in other environments, should we move it out into code?